plugins/gatsby-source-iiif/generateMD.js

```diff
@@ -5,7 +5,7 @@ const configuration = require('../../content/configuration.js')
 
 const getMDFile = (manifest) => {
   const mdFile = `---
-title: "${manifest.label[configuration.languages.default]}"
+title: "${manifest.label[configuration.siteMetadata.languages.default]}"
```

Comment:
If you're making this switch, can you remove line 42 from `content/configuration.js` as well?